docs/source/markdown/podman-secret-ls.1.md.in

```diff
@@ -28,6 +28,20 @@ Valid filters are listed below:
 
 Format secret output using Go template.
 
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder**          | **Description**                                                   |
+| =====------------------- | ----------------------------------------------------------------- |
```

Comment:
aaaaaannd, you fixed one, but introduced equals in another.

Ninety-nine little bugs in the code
Ninety-nine little bugs
Fix one bug
commit & re-push
One hundred little bugs in the code

Reply:
I wonder if Emacs is doing something strange here...  Or just PEBCAK